<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMSysApp</class>
 <widget class="QWidget" name="SMSysApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="top" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>6</y>
     <width>850</width>
     <height>64</height>
    </rect>
   </property>
   <widget class="QWidget" name="icon" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>7</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbTitle">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>管理您的手机预装软件</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSubTitle">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>您可以在此进行禁用,卸载和关闭开机自启动操作(此功能需要roo您的手机), 一键Root</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbShuaxin">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbHuanyuan">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>850</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
